lib/presentation/pages/intro_page.dart

```diff
@@ -75,7 +76,11 @@ class _IntroPageState extends State<IntroPage> {
     return Scaffold(
       body: PopScope(
         canPop: state == IntroState.welcome,
-        onPopInvoked: (_) => previousPage(),
+        onPopInvoked: (_) {
+          if (!onTapReadyConfetti) {
+            previousPage();
+          }
```

Comment:
This statement should move to inside `previousPage` method for the functionality to effect also the back button